packages/bot-web-ui/src/components/journal.jsx

```diff
@@ -1,97 +1,89 @@
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
+import ContentLoader from 'react-content-loader';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
 import { Checkbox, Icon, ThemedScrollbars } from '@deriv/components';
-import { localize } from '@deriv/translations';
+import { localize, Localize } from '@deriv/translations';
 import { message_types } from '@deriv/bot-skeleton';
+import { log_types } from '@deriv/bot-skeleton/src/constants/messages';
+import { contract_stages } from '../constants/contract-stage';
 import { connect } from '../stores/connect';
 import '../assets/sass/journal.scss';
 
 const DateItem = ({ date, time }) => {
     return (
-        <div>
-            <span className='journal__table--date'>{date}</span>
-            <span className='journal__table--time'>{time}</span>
-        </div>
+        <>
+            <span className='journal__text-date'> {date} </span> | <span className='journal__text-time'> {time} </span>
+        </>
     );
 };
 
-const FormatMessage = ({ message }) => {
-    if (React.isValidElement(message)) {
-        return message;
-    }
-    if (typeof message === 'object') {
-        return JSON.stringify(message);
-    }
-    if (typeof message !== 'string') {
-        return message;
-    }
-
-    // TODO: Below won't work for localised strings.
-    const key_words = ['Bought', 'Sold', 'Profit amount', 'Loss amount'];
-    const messages = message.split(':');
-
-    if (messages.length < 2) {
-        return message;
-    }
-
-    const title = messages[0];
-    const value = messages.slice(1).join(':');
-
-    let title_color, value_color;
-
-    switch (title) {
-        case key_words[0]: {
-            // Bought
-            title_color = 'info';
-            break;
-        }
-        case key_words[1]: {
-            // Sold
-            title_color = 'error';
-            break;
-        }
-        case key_words[2]: {
-            // Profit amount
-            value_color = 'success';
-            break;
-        }
-        case key_words[3]: {
-            // Loss amount
-            value_color = 'error';
-            break;
-        }
-        default: {
-            title_color = value_color = undefined;
+const FormatMessage = ({ logType, className, extra }) => {
+    const getLogMessage = () => {
+        switch (logType) {
+            case log_types.LOAD_BLOCK: {
+                return localize('Blocks are loaded successfully');
+            }
+            case log_types.NOT_OFFERED: {
+                return localize('Resale of this contract is not offered.');
+            }
+            case log_types.PURCHASE: {
+                const { longcode, transaction_id } = extra;
+                return (
+                    <Localize
+                        i18n_default_text='<0>Bought</0>: {{longcode}} (ID:{{transaction_id}})'
```

Comment:
Should there be a space between `ID` and `transaction_id`?

Reply:
it should. thanks 👍